### DISCUSSION TITLE
Searching for Nordic format addresses

### DISCUSSION THREAD

**Discussion by styrheim** [osmandapp/OsmAnd#15999] [General]
Osmand+ 4.3.3

In Tórshavn, a search for "Tinghúsvegur 72" finds nothing, not even the street.
The English format search works: "72 Tinghúsvegur", finding both street and house.

I would like a search for Nordic format addresses to word as well.

**Comment by tommycrock** [comment]
Strange, it works for me
![Screenshot_20221217_080701_OsmAnd+](https://user-images.githubusercontent.com/5749884/208232428-3e5855e1-8234-4f06-b0fe-9d0b23d0d75f.jpg)

**Comment by styrheim** [comment]
Works for me too, now. The only change, is that I have restarted the phone. 
But I'm quite sure I tested the same search string, as it came via an event location I have entered into Google Calendar.

With no way to reproduce, I guess this "did not happen".
